Sources/NIOFileSystem/BufferedReader.swift

```diff
@@ -239,3 +239,6 @@ extension ReadableFileHandleProtocol {
         BufferedReader(wrapping: self, initialOffset: initialOffset, capacity: Int(capacity.bytes))
     }
 }
+
+@available(*, unavailable)
+extension BufferedReader: Sendable {}
```

Comment:
@glbrntt Should this be conditionally `Sendable` based on `Handle`?

Reply:
Yes